src/dao.cpp

```diff
@@ -817,12 +917,7 @@ namespace hypha
       //role.getOrFail(DETAILS, MIN_TIME_SHARE)->getAs<int64_t>();
               
       bool checkNewTimeShareMin = true;
-      
-      if (modifier == common::MOD_WITHDRAW) 
-      {
-        //checkNewTimeShareMin = false;
-      }
-      
+            
       if (checkNewTimeShareMin) 
```

Comment:
This is always true, is it expected? It was always true before this PR anyway.

Reply:
Yep, it's expected to be true, it was variable before, but now I can remove it.